pkg/util/util.go

```diff
@@ -25,24 +25,27 @@ const (
 var ChartNamespace string
 
 func GetEncryptionTransformers(encryptionConfigSecretName string, secrets v1core.SecretController) (map[schema.GroupResource]value.Transformer, error) {
-	var transformerMap map[schema.GroupResource]value.Transformer
 	// EncryptionConfig secret ns is hardcoded to ns of controller in chart's ns
 	// kubectl create secret generic test-encryptionconfig --from-file=./encryption-provider-config.yaml
 	logrus.Infof("Get encryption config from namespace %v", ChartNamespace)
 	encryptionConfigSecret, err := secrets.Get(ChartNamespace, encryptionConfigSecretName, k8sv1.GetOptions{})
 	if err != nil {
-		return transformerMap, err
+		return nil, err
 	}
 	encryptionConfigBytes, ok := encryptionConfigSecret.Data[encryptionProviderConfigKey]
 	if !ok {
-		return transformerMap, fmt.Errorf("no encryptionConfig provided")
+		return nil, fmt.Errorf("no encryptionConfig provided")
 	}
 	err = ioutil.WriteFile(encryptionProviderConfigKey, encryptionConfigBytes, os.ModePerm)
 	defer os.Remove(encryptionProviderConfigKey)
+
+	// ctx context.Context, filepath string, reload bool, apiServerID string
```

Comment:
Probably intended to be a few lines lower? Or just a temporary dev note?

Reply:
fixed